### ISSUE TITLE
Multicolor font support. This is a new OpenType feature that has four different implementations. Cur...

### ISSUE THREAD

**Issue by Fyrd** [Fyrd/caniuse#1018]
Multicolor font support. This is a new OpenType feature that has four different implementations. Currently two of them are supported in Firefox, one in IE11 on Windows 8.1+.

See http://pixelambacht.nl/2014/multicolor-fonts/#testing-your-browser

---------
##### This issue was imported from [Google Moderator](https://www.google.com/moderator/)
Moderator votes: __+18__
Issue added by __Roel N__ on __2014-10-26__
To vote this issue up or down, simply include `+1` or `-1` in your comment.

**Comment by pkuczynski** [comment]
Seems like only very few browsers and only on selected platforms/os support this type of fonts. There are also couple of proposals how to achieve those and its not in any way standardised.

* http://caniemoji.com
* https://code.google.com/p/chromium/issues/detail?id=333011
* https://bugzilla.mozilla.org/show_bug.cgi?id=889401

@Fyrd shall we add it or not?

**Comment by valtlait** [comment]
+1

**Comment by rotsee** [comment]
+1

**Comment by svgeesus** [comment]
> Seems like only very few browsers and only on selected platforms/os support this type of fonts. 

True, but support is improving so this is a good candidate for caniuse. Currently Firefox supports SVG and COLR; Safari supports SBIX, Chrome supports nothing (for Webfonts; CBDT/CBLC is supported for installed fonts only) and Edge recently jumped from COLR/CPAL only (Edge <= 13) to supporting SBIX, SVG, and CBDT/CBLC as well with the Windows 10 update and Edge 14.

> There are also couple of proposals how to achieve those and its not in any way standardised.

That is incorrect. There are four ways to do this; three are standardized by ISO and are part of the OpenType v.3 specification. The fourth, SBIX, is undocumented by Apple but has been reverse engineered by third parties.

**Comment by svgeesus** [comment]
Oh and +1

**Comment by RoelN** [comment]
Shameless +1 even though I added this issue :)

**Comment by tphinney** [comment]
+1

Writing this because I wanted to get the latest info from CanIUse, and it wasn't there.

The previously undocumented SBIX flavor was added to OpenType 1.8, launched in October 2016. All four flavors are now formally part of OpenType and standardized by ISO in the Open Font Format.

**Comment by Ugoku** [comment]
+1

**Comment by Jolg42** [comment]
+1 👍

**Comment by wasd171** [comment]
+1

**Comment by tobireif** [comment]
+1

**Comment by tobireif** [comment]
Some browser support info is at https://www.colorfonts.wtf/ → "Web browsers".

**Comment by Schweinepriester** [comment]
+1

**Comment by djrrb** [comment]
+1

**Comment by chrishanel** [comment]
+1

**Comment by yisibl** [comment]
https://www.colorfonts.wtf/#w-node-85d8080e63a6-0134536f

![image](https://user-images.githubusercontent.com/2784308/107009666-50b79880-67d0-11eb-9698-bc87cf81b50e.png)

**Comment by yisibl** [comment]
+1

**Comment by Fyrd** [comment]
COLR/CPAL support now available at https://caniuse.com/colr

**Comment by weiweihuanghuang** [comment]
+1 for SBIX support

**Comment by t3chguy** [comment]
Worth noting that Safari 17 breaks COLR support

16.5
![image](https://github.com/Fyrd/caniuse/assets/2403652/9a245926-1045-45e2-86b0-93a376f33fe6)

17.0
![image](https://github.com/Fyrd/caniuse/assets/2403652/599b3e09-c493-47e3-8a6c-5409a911e2b0)

**Comment by Schweinepriester** [comment]
@t3chguy is there a webkit bug for this already?

**Comment by t3chguy** [comment]
Not that I could see

**Comment by Schweinepriester** [comment]
Trying to summon @jensimmons regarding this^, COLR(v0) support in Safari 17? 0:-)

**Comment by jensimmons** [comment]
@t3chguy Could you file an issue at bugs.webkit.org? That would be helpful.

**Comment by t3chguy** [comment]
@jensimmons https://bugs.webkit.org/show_bug.cgi?id=262223
